src/mlmc/quantity_estimate.py

```diff
@@ -51,13 +52,18 @@ def estimate_mean(quantity, chunk_size=512000000):
                 # level_chunk is Numpy Array with shape [M, chunk_size, 2]
                 n_samples[level_id] += chunk.shape[1]
                 n_rm_samples[level_id] += n_mask_samples
+
+                # No samples in chunk
+                if chunk.shape[1] == 0:
```

Comment:
That should not happen.

Reply:
It could happen. For example, if we are subsampling, a chunk might be empty.